test/asm/sym-scope.err

```diff
@@ -1,9 +1,3 @@
 error: sym-scope.asm(4):
-    Local label '.tooSoon' in main scope
-error: sym-scope.asm(5):
-    Local label 'Nice.try' in main scope
-error: sym-scope.asm(17):
-    Not currently in the scope of 'Parentheses'
-error: sym-scope.asm(21):
-    Not currently in the scope of 'Parent'
-error: Assembly aborted (4 errors)!
+    Unqualified local label '.tooSoon' in main scope
+error: Assembly aborted (1 error)!
```

Comment:
Aborting on line 4 means all those subsequent lines don't even get run. And local-without-parent.asm covers this possibility. So this line should be deleted.

Reply:
It looks like you're asking to repurpose `sym-scope.asm` from testing which lines do and don't produce an error to testing the output of a well-formed source file. Do I understand correctly?

Reply:
I'm saying, delete the local-wrong-parent.asm test. It is now a redundant misnomer of a test.

Then delete this line you added to sym-scope.asm, which causes the "Unqualified local label" error, because it causes the assembly to be aborted before the rest of the test file can run.

Reply:
Thanks for clarifying. I believe I did that in the most recent commit.